Sources/MongoSwift/ReadPreference.swift

```diff
@@ -202,6 +202,16 @@ public struct ReadPreference: Equatable {
                 )
             }
         }
+        if let tagSets = tagSets {
+            if mode == .primary {
+                guard tagSets == [BSONDocument()] else {
+                    throw MongoError.InvalidArgumentError(
+                        message: "Invalid \(MONGOC_URI_READPREFERENCETAGS) \(tagSets): " +
```

Comment:
Can we update this to use `MongoConnectionString.OptionName.readPreferenceTags`? This way when we eventually stop using libmongoc we won't need to change this message. The `OptionName` enum is currently `fileprivate` so you'll need to update it to `internal`.

Reply:
sure